### ISSUE TITLE
Discrepancy between yards_gained and yardline_100

### ISSUE THREAD

**Issue by vale-tech** [mrcaseb/nflfastR#96]
This looks like an issue with the NFL data, but posting in case it helps. 

game_id: 2019_07_NO_CHI
play_id(s): 3186, 3165

The pass play was noted as 8 yards, starting from CHI 29. The following play is from CHI 33 (4 yards). The pass catcher has 4 additional yac, air_yards, etc. when compared to AirYards.com data.

**Comment by mrcaseb** [comment]
I do not see this problem. Is it possible that you have reversed the order of the plays or misunderstood the meaning of the variables?

1. play_id 3165 started at CHI 33 and gained 4 yards. The next play is play_id 3186

1. play_id 3186 started at CHI 29. 

That's correct.

```r
library(tidyverse)
nflfastR:::load_pbp(2019) %>% 
  filter(game_id == "2019_07_NO_CHI", between(play_id, 3100, 3300)) %>% 
  select(game_id, play_id, yards_gained, yardline_100, yrdln, air_yards, yards_after_catch, complete_pass)
```

```
# A tibble: 9 x 8                                                                                                               
  game_id        play_id yards_gained yardline_100 yrdln  air_yards yards_after_catch complete_pass
  <chr>            <dbl>        <dbl>        <dbl> <chr>      <dbl>             <dbl>         <dbl>
1 2019_07_NO_CHI    3115            6           62 NO 38         -5                11             1
2 2019_07_NO_CHI    3144           23           56 NO 44         NA                NA             0
3 2019_07_NO_CHI    3165            4           33 CHI 33        NA                NA             0
4 2019_07_NO_CHI    3186            8           29 CHI 29         4                 4             1
5 2019_07_NO_CHI    3210            0           21 CHI 21        21                NA             0
6 2019_07_NO_CHI    3232           15           21 CHI 21        NA                NA             0
7 2019_07_NO_CHI    3253            2            6 CHI 6         NA                NA             0
8 2019_07_NO_CHI    3274            4            4 CHI 4         -4                 8             1
9 2019_07_NO_CHI    3299            0           15 CHI 15        NA                NA             0
```

I will close this as it seems to be a misinterpretation of either `play_id` , `yardline_100` or `yards_gained`

**Comment by vale-tech** [comment]
Well now the air yards is 4. When I filed the issue, the air yards was 8, which was not correct. If your data is constantly shifting around then I don't know what to say. Thanks for looking.

**Comment by mrcaseb** [comment]
> 
> 
> Well now the air yards is 4. When I filed the issue, the air yards was 8, which was not correct. If your data is constantly shifting around then I don't know what to say. Thanks for looking.

It's possible this is due to the NFL going back and fixing some stuff. But since it's from the 2019 season I would be surprised if the air_yards changed but I don't know when you downloaded your data. However, please do not feel offended, it was not meant that way.

**Comment by vale-tech** [comment]
Yeah, it was a bit surprising to me as well. Which is why I checked what the data looked like in Gamecenter for those plays and made sure that the data that was being downloaded with the db function matched the static files, but 20ish days or so ago both had 8 air yards as a result of this play so I filed an issue. I tried very hard to double check everything on my end before opening one as I understand it would be quite time consuming to track all of these things down.
